pkg/connector/sso_group.go

```diff
@@ -249,6 +254,9 @@ func (g *ssoGroupResourceType) Revoke(ctx context.Context, grant *v2.Grant) (ann
 	})
 	if err != nil {
 		l.Error("aws-connector: Failed to delete group membership", zap.Error(err))
+		if strings.Contains(err.Error(), "https response error StatusCode: 400") {
+			return nil, nil
+		}
```

Comment:
This is unlikely to be correct. The error code here should be 404 if something is not found. 400 means bad request like we sent something incorrectly.